### PR TITLE
chain management 

### PR DESCRIPTION
**Pull request by rabbyOnTheMars**

### CODE REVIEW

File: src/ui/component/ChainSelector/Modal.tsx [hunk 1]
```diff
@@ -61,49 +70,37 @@ const ChainSelectorModal = ({
 
   return (
     <Modal
-      width="360px"
+      width="400px"
```
Comment (vvvvvv1vvvvvv) [inline src/ui/component/ChainSelector/Modal.tsx:73]: try use https://ant.design/components/drawer/ with animation

**Comment by vvvvvv1vvvvvv** [comment]
image
should be better to remove the scrollbar

File: src/ui/component/ChainSelector/Modal.tsx (hunk 1 above)
Comment (rabbyOnTheMars) [reply src/ui/component/ChainSelector/Modal.tsx:78]: done

File: src/ui/component/ChainSelector/Modal.tsx [hunk 2]
```diff
@@ -40,10 +38,23 @@ const ChainSelectorModal = ({
...
+      state: {
+        connection,
+      },
+    });
+  };
   const init = async () => {
-    setEnableChains(await wallet.getEnableChains());
-    setCurrentAccount(await wallet.syncGetCurrentAccount());
+    const savedChains = await wallet.getSavedChains();
+    const getSupportChains = await wallet.getSupportChains();
```
Comment (vvvvvv1vvvvvv) [inline src/ui/component/ChainSelector/Modal.tsx:51]: Why we need supportChains?
This should be all chains right?

```typescript
import { CHAINS } from 'consts';
Object.values(CHAINS)
```

Comment (rabbyOnTheMars) [reply src/ui/component/ChainSelector/Modal.tsx:51]: So we don't  need to retrieve  chain from open api, but from local ?

Comment (rabbyOnTheMars) [reply src/ui/component/ChainSelector/Modal.tsx]: done

File: src/ui/views/ChainManagement/index.tsx [hunk 3]
```diff
@@ -10,23 +11,21 @@ import './style.less';
 export const ChainManagementList = () => {
   const wallet = useWallet();
   const { t } = useTranslation();
-  const [allChains, setAllChains] = useState<Chain[]>([]);
-  const [chains, setChains] = useState<Chain[]>([]);
+  const [allChains, setAllChains] = useState<Chain[]>(Object.values(CHAINS));
```
Comment (vvvvvv1vvvvvv) [inline src/ui/views/ChainManagement/index.tsx:14]: A constant no need to set as state right?

Comment (rabbyOnTheMars) [reply src/ui/views/ChainManagement/index.tsx]: done

**Review by vvvvvv1vvvvvv — APPROVED** [review]

**Pull request merged**